### DISCUSSION TITLE
my school admin might be finding out about skiovox (??)

### DISCUSSION THREAD

**Discussion by brobcoolguy** [3kh0/ext-remover#923] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
so earlier today i opened my chromebook which was in skiovox at the moment and I remember there being a message that was something like:

"(schoolorganization).org is forcing you to restart your chromebook by (date)." or something like that, it had the same gui as when you would sign out or something.

so i just skipped past it not really thinking about it untill my chromebook suddenly restarts (this has happened a few times, i dont know if it was correlated to the beforementioned warning), and im back to the sign in screen. so i go back to the kiosk apps menu to activate skiovox again and i notice something weird...

the main kiosk app i would use to use skiovox has now disappeared. and im like wtf and i call my friend asking him if this happened to him too, weird thing is... he still has the app, but i dont. (note: im probally the only person in my school/class that knows about and uses skiovox currently, my friend has never used it)

luckily theres one more app that works with skiovox but now im a bit concerned, do you guys think theres a way i could make sure this doesnt happen again or smth?

**Comment by PenguinLord561** [comment, marked as answer]
do a caub

**Reply by brobcoolguy** [reply]
already done that, thx
